### PR TITLE
Added the ability to set the default database to Set-DbaLogin

### PR DESCRIPTION
**Pull request by mikefrobbins**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [X] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

Adds the ability to set the default database with the Set-DbaLogin command.

Adds a DefaultDatabase parameter and help based off of the DefaultDatabase parameter from New-DbaLogin.

See the help for how to use this new parameter.

image

The databases variable contains a list of databases on the specified SQL Server. The command shown in the image sets the default database to master for all users who are set to a database that does not exist.

### CODE REVIEW

File: functions/Set-DbaLogin.ps1 [hunk 1]
```diff
@@ -152,6 +160,8 @@ function Set-DbaLogin {
         [string[]]$Login,
         [Alias("Password")]
         [object]$SecurePassword, #object so that it can accept credential or securestring
+        [Alias("DefaulDB")]
```
Comment (shaneis) [inline functions/Set-DbaLogin.ps1]: I only say this cause I was looking at typos and saw the tweet at the same time. You're missing a 't' in 'DefaultDB'
```suggestion
        [Alias("DefaultDB")]
```

**Comment by potatoqualitee** [comment]
Ahh, there is one failure, Mike. We use the OTSB style which is set for VS Code users, but not if you use another editor (so i understand, i use another editor ;). You can execute `Invoke-DbatoolsFormatter` against  `Set-DbaLogin.ps1`

Also, I'm not sure why the Set-DbaLogin.Tests.ps1 passed, the parameter unit tests must be updated. I can do both of these if you like, Mike. Otherwise, please update and I'll merge on pass tests 👍

File: functions/Set-DbaLogin.ps1 (hunk 1 above)
Comment (mikefrobbins) [reply functions/Set-DbaLogin.ps1]: I wanted to make sure the spelling and casing of the parameter and its alias matched the rest of the project so I copy and pasted it from New-DbaLogin which evidently has the same typo.

**Comment by mikefrobbins** [comment]
Requested changes submitted.

**Comment by potatoqualitee** [comment]
awesome, thank you! also good to note about the single file, i'll keep that in mind when i write about vs code.

**Comment by shaneis** [comment]
Just checked New-DbaLogin and you're 100% right 😱

I'll fix it when I get to a computer if nobody wants to fix it now 👍

Thanks for the heads up and the PR!

**Comment by potatoqualitee** [comment]
@shaneis - PR from Mike is in and merged! 💯

**Pull request merged**